src/__tests__/02e-onchain-send.ts

```diff
@@ -165,6 +243,40 @@ it("makesOnchainOnUsTransaction", async () => {
   }
 })
 
+it("makesOnchainOnUsSendAllTransaction", async () => {
+  try {
```

Comment:
do we need a try catch here?

Reply:
the test might enter in the catch block without failing. I would either remove the try catch and let the test fails, or add a `fail()` comment in the catch block if this is needed for some reasons.

other the test could have an issue and would be reported as failed by jest

Reply:
Makes sense. That test was based on the 'makesOnchainOnUsTransaction' test which also has the try. Will remove it from both.